connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/User.java

```diff
@@ -0,0 +1,162 @@
+package com.redhat.cloud.notifications.connector.email.model;
+
+import io.vertx.core.json.JsonObject;
+
+import java.util.Objects;
+
+/**
+ * Represents the internal user model used to effectively unify and process the
+ * users from the user providers.
+ */
+public final class User {
+
+    private String id;
+    private String username;
+    private String email;
+    private String firstName;
+    private String lastName;
+    private boolean isActive;
+    private boolean isAdmin;
+
+    public User() { }
+
+    public User(
+        String id,
+        String username,
+        String email,
+        String firstName,
+        String lastName,
+        boolean isActive,
+        boolean isAdmin
+    ) {
+        this.id = id;
+        this.username = username;
+        this.email = email;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.isActive = isActive;
+        this.isAdmin = isAdmin;
+    }
+
+    /**
+     * Builds a user from the JSON RBAC User object.
+     * @param rbacUser the RBAC User JSON object.
+     */
+    public User(final JsonObject rbacUser) {
+        this.id = rbacUser.getString("id");
+        this.username = rbacUser.getString("username");
+        this.email = rbacUser.getString("email");
+        this.firstName = rbacUser.getString("first_name");
+        this.lastName = rbacUser.getString("lastName");
+        this.isActive = rbacUser.getBoolean("is_active");
+        this.isAdmin = rbacUser.getBoolean("is_org_admin");
+    }
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (o == null || this.getClass() != o.getClass()) {
+            return false;
+        }
+
+        final User user = (User) o;
+
+        if (this.isActive != user.isActive) {
+            return false;
+        }
+
+        if (this.isAdmin != user.isAdmin) {
+            return false;
+        }
+
+        if (!this.id.equals(user.id)) {
+            return false;
+        }
+
+        if (!this.username.equals(user.username)) {
+            return false;
+        }
+
+        if (!this.email.equals(user.email)) {
+            return false;
+        }
+
+        if (!Objects.equals(this.firstName, user.firstName)) {
+            return false;
+        }
+
+        return Objects.equals(this.lastName, user.lastName);
```

Comment:
```suggestion
        if (this == o) {
            return true;
        }
        if (o instanceof User) {
            User other = (User) o;
            return Objects.equals(isActive, other.isActive) &&
                    Objects.equals(isAdmin, other.isAdmin) &&
                    Objects.equals(id, other.id) &&
                    Objects.equals(username, other.username) &&
                    Objects.equals(email, other.email) &&
                    Objects.equals(firstName, other.firstName) &&
                    Objects.equals(lastName, other.lastName);
        }
        return false;
```

Reply:
Not needed since we got rid of the `User` class.